draft-ietf-scitt-architecture.md

```diff
@@ -199,7 +199,7 @@ Artifact:
 
 Auditor:
 
-: an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service.
+: an entity that can review entries of a Transparency Service's Append-only Log to prove the application and enforcment of Registration Policy during each Registration process step.
```

Comment:
To date, it was my understanding we were referring to an Auditor as someone that verifies the correctness of the log, that no entries were added, modified or removed. 
This sounds like an auditor would also be required to assure that any referenced registration policy was adhered to.

Reply:
In the current state of the draft around the capabilities of a TS to publish the registration policy and no structured mechanism for an Auditor to understand it, it is unclear how this ability recommended in this change is possible.

How would an Auditor perform registration policy enforcement in actuality?

Reply:
Actually, the information on the ledger is what the Auditor reviews.  The confusion of auditors checking the consistency of the "Merkle tree" is left as obvious functionality or in the RFC on Merkle or Blockchains.  The selection of which is left to the implementors.  I am fine with Steve's suggestion.

AJ The structure of the data on the ledger is explicitly not covered as it varies based on identity technology used.  In a CA issued x.509 it will be different than in a DID-WebXYZ one.  

The mechanism to configure "trusted roots" (registration policy) is also not standardize and can be left out.  If there is no configured identity providers, then there should be no receipts handed out.  The interesting problem for the future is detecting in a possible federated world when that policy changes.  This is where we were hung up for months and decided to simply scale back.

Reply:
This suggested change is accesptable